lib/src/json_schema/json_schema.dart

```diff
@@ -76,24 +76,34 @@ class JsonSchema {
   }
 
   JsonSchema._fromRootMap(this._schemaMap, SchemaVersion schemaVersion,
-      {Uri fetchedFromUri, bool isSync = false, Map<String, JsonSchema> refMap, RefProvider refProvider}) {
+      {Uri fetchedFromUri,
+      bool isSync = false,
+      Map<String, JsonSchema> refMap,
+      RefProvider refProvider,
+      Map<Uri, bool> metaschemaVocabulary}) {
```

Comment:
Comma here will help the formatter make this look better:
```suggestion
      Map<Uri, bool> metaschemaVocabulary,}) {
```